examples/DrawingInTheAir/DrawingInTheAir.ino

```diff
@@ -166,10 +169,10 @@ void readVectorFromIMU(byte vector[])
     unsigned int i = 0;
 
     /* Wait until button is pressed */
-    while (digitalRead(buttonPin) == HIGH);
+    while (digitalRead(buttonPin) == LOW);
```

Comment:
Why did you reverse the logic?
Are we assuming that the user would pull the gpio low with a pull-down resistor?
Then a button press would connect the gpio to 3.3V? 

I actually think that the original implementation is better because user can simply use pinMode(pin, INPUT_PULLUP) instead of requiring a pull-up resistor